pegasus/sites.v3/code.org/public/educate/csf.haml

```diff
@@ -97,7 +97,7 @@ theme: responsive_full_width
 
 %section
   .wrapper
-    = view :curriculum_pages_pl_section, facilitator_desc: hoc_s(:professional_learning_section_block_facilitator_led_workshops_csf), apply_url: "/professional-development-workshops", modules_url: CDO.studio_url("/s/k5-onlinepd-2023")
+    = view :curriculum_pages_pl_section, facilitator_desc: hoc_s(:professional_learning_section_block_facilitator_led_workshops_csf), apply_url: "/professional-development-workshops", modules_url: CDO.studio_url("/s/k5-onlinepd")
```

Comment:
Unfortunately this is one of the very weird edge cases that has to stay with a year because we haven't cleaned up our course system fully

Reply:
No worries, just updated these two instances!